### ISSUE TITLE
Chinese character not supported

### ISSUE THREAD

**Issue by catelinn** [tabulapdf/tabula#638]
Hello this program won't work with pdf with non English characters. Will there be language support in the future?

**Comment by jeremybmerrill** [comment]
Yes, it works with many non-English PDFs, including many with Chinese characters. If you send your PDF, we can try to take a look at it; it's likely a more nuanced problem.
